src/idsk/components/table-filter/table-filter.js

```diff
@@ -0,0 +1,235 @@
+import '../../../govuk/vendor/polyfills/Function/prototype/bind'
+import '../../../govuk/vendor/polyfills/Event' // addEventListener and event.target normaliziation
+import {toggleClass} from "../../common";
+
+
+function TableFilter($module) {
+  this.$module = $module
+  this.selectedFitlersCount = 0
+  this.$activeFilters = []
+}
+
+TableFilter.prototype.init = function () {
+  // Check for module
+  var $module = this.$module
+  if (!$module)
+    return
```

Comment:
ja odporucam urcite aj vsade pouzivat zatvorky {} aj ked je to len jeden riadok, obzvlast v jsku